### PR TITLE
[pacemaker] Add log file.

### PR DESCRIPTION
**Pull request by psachin**
sosreport failed to include pacemaker.log file in one of the case.
I don't know if it is appropriate to give a case number here in public.

Signed-off-by: Sachin Patil <psachin@redhat.com>

### CODE REVIEW

**Comment by bmr-cymru** [comment]
> I don't know if it is appropriate to give a case number here in public.

There's no need to mention things like this upstream - just an explanation of the problem and how it is fixed.

File: sos/plugins/pacemaker.py [hunk 1]
```diff
@@ -26,6 +26,11 @@ def setup(self):
         self.add_copy_spec([
             "/var/lib/pacemaker/cib/cib.xml"
         ])
+
+        self.limit = self.get_option("log_size")
```
Comment (bmr-cymru) [inline sos/plugins/pacemaker.py]: Why define an instance variable for something that is used exactly once?

Just make the call in-line at the site of `self.add_copy_spec_limit()`.

File: sos/plugins/pacemaker.py [hunk 2]
```diff
@@ -26,6 +26,11 @@ def setup(self):
...
+        self.add_copy_spec_limit("/var/log/pacemaker.log",
```
Comment (bmr-cymru) [inline sos/plugins/pacemaker.py]: Is this not collected by `pacemaker`'s own `crm_report`?

**Comment by psachin** [comment]
I did not see pacemaker.log file inside `sosreport_log/sos_commands/cluster/crm_report/` when I generated sosreport using

`sosreport -o cluster`

**Comment by psachin** [comment]
This is also confirmed by one of our associate - Matt Flusche.

**Comment by bmr-cymru** [comment]
Please talk to Shane Bradley - @sbradley7777 - he's currently working on a refactor and cleanup of the cluster support and is in the best position to know what's being done by `crm_report` and what should be handled in `sos`.

**Comment by sbradley7777** [comment]
We are currently doing a refactor of cluster plugin and you can view it here: 

- https://github.com/sosreport/sos/issues/587
- https://github.com/sosreport/sos/commit/44fff9d258325dc738cc36f300bc3de577cc4a59

The pacemaker.log will be collected by the pacemaker.log in refactor version (current upstream version does not collect). 

This issue should be fixed if the above branch is merged into upstream.

**Comment by psachin** [comment]
Great! Thanx @sbradley7777

**Comment by bmr-cymru** [comment]
As Shane's work is going to include this file I will close this pull request out for now - let us know if anything else is needed.

**Pull request closed without merging**